src/pages/my_page/user_profile/MyFieldAndBackground.tsx

```diff
@@ -20,6 +25,9 @@ export const MyFieldAndBackground = () => {
   const user = useSelector((store: RootState) => store.user);
   const personId = user?.cristinId ?? '';
 
+  const [keywordSearchTerm, setKeywordSearchTerm] = useState('');
+  const debouncedSearchTerm = useDebounce(keywordSearchTerm);
```

Comment:
gir mening å følge opp resten også når vi renamer kilden
```suggestion
  const debouncedKeywordSearchTerm = useDebounce(keywordSearchTerm);
```